Periodic_4_hyperbolic_triangulation_2/package_info/Periodic_4_hyperbolic_triangulation_2/description.txt

```diff
@@ -0,0 +1,3 @@
+Package 2D periodic hyperbolic triangulation: provides Delaunay triangulations
+of the Bolza surface, represented as periodic triangulations in the Poincaré disk 
+model of the hyperbolic plane, with demos, examples, and tests.  
```

Comment:
Both HT2 and P4HT2 are missing the long_descriptor file

Reply:
I've added this file to both packages.